test/suites/basic.sh

```diff
@@ -423,7 +423,7 @@ test_basic_usage() {
   lxc profile create unconfined
   lxc profile set unconfined security.privileged true
   lxc init testimage foo2 -p unconfined -s "lxdtest-$(basename "${LXD_DIR}")"
-  [ "$(stat -L -c "%a" "${LXD_DIR}/containers/foo2")" = "700" ]
+  [ "$(stat -L -c "%a" "${LXD_DIR}/containers/foo2")" = "100" ]
```

Comment:
@stgraber the introduction of `CreateMountPath` above has resulted in this test needing to be changed.

Reply:
@stgraber this change has also broken tests for the old storage drivers that are creating them with 700. Shall I update them to create as 100?

Reply:
Yes, that's probably the easiest. We already have a check in place to validate the 100 mode later on (in security) but this test is running before that logic ever applies :)

Reply:
@stgraber thanks